<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="VINU" name="CCW" tests="3" failures="1" timestamp="17 May 2015 21:06:42 GMT" time="0.032" errors="0">
  <testcase name="TC1" time="0.012" classname="Test.TestSet1"/>
  <testcase name="TC2" time="0.0" classname="Test.TestSet1">
    <skipped/>
  </testcase> <!-- TC2 -->
  <testcase name="TC_3" time="0.0" classname="Test.TestSet1">
    <failure type="java.lang.AssertionError" message="expected [BAD] but found [Good]">
      <![CDATA[java.lang.AssertionError: expected [BAD] but found [Good]
	at Test.TestSet1.TC_3(TestSet1.java:22)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- TC_3 -->
</testsuite> <!-- CCW -->
